irctest/server_tests/buffering.py

```diff
@@ -86,10 +86,10 @@ def testNoTags(self, sender_function, colon):
             if messages and ERR_INPUTTOOLONG in (m.command for m in messages):
                 # https://defs.ircdocs.horse/defs/numerics.html#err-inputtoolong-417
                 self.assertGreater(
-                    len(line + payload + "\r\n"),
+                    len((line + payload + "\r\n").encode()),
```

Comment:
This is the actual bug fix, everything else is either refactoring or additional assertions